doc/man7/EVP_CIPHER-ASCON128.pod

```diff
@@ -0,0 +1,50 @@
+=pod
+
+=head1 NAME
+
+EVP_CIPHER-ASCON128 - The Ascon128 EVP_CIPHER implementations
+
+=head1 DESCRIPTION
+
+Support for ASCON-128 symmetric encryption using the B<EVP_CIPHER> API
+
+=head2 Algorithm Names
+
+The following algorithms are available in the default provider:
+
+=over 4
+
+=item "ASCON128"
```

Comment:
This name should be Ascon-AEAD128 as specified in SP800-232? (This is not one of ASCON-128, ASCON-128a, or ASCON-80pq)